Dockerfile

```
@@ -5,6 +5,13 @@
 # Base Image
 FROM ruby:2.3.1-onbuild
 
+# Update git client
+RUN echo "deb http://ftp.us.debian.org/debian/ buster main contrib non-free" >> /etc/apt/sources.list \
+    && apt-get update \
+    && apt-get remove  -y binutils --force-yes \
```

Please remove the extra space between `remove` and `-y` on this line:
```suggestion
    && apt-get remove -y binutils --force-yes \
```